File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-06/plagiarized/L4/06/TestSatuArray.kt
package IR

import java.util.*

object TestSatuArray {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        var arr = IntArray(10) //inisialisasi array
        arr = inputArr(arr)
        printArr(arr)
    }

    fun inputArr(arr: IntArray): IntArray {
        val sc = Scanner(System.`in`)
        for (x in 0..9) { //looping untuk input array
            print("Read a number: ")
            arr[x] = sc.nextInt() //input ke array
        }
        return arr
    }

    fun printArr(arr: IntArray) {
        for (x in 9 downTo 0) { //looping untuk menampilkan array
            println(arr[x]) //menampilkan array
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L4/06/TestSatuArray.py
import sys

def inputArr(arr):
    for x in range(10):
        print("Read a number: ", end="")
        arr[x] = int(input())
    return arr

def printArr(arr):
    for x in range(9, -1, -1):
        print(arr[x])

def main():
    arr = [0] * 10
    arr = inputArr(arr)
    printArr(arr)

if __name__ == "__main__":
    main()

